common/djangoapps/third_party_auth/models.py

```diff
@@ -29,6 +29,21 @@
 
 from .lti import LTI_PARAMS_KEY, LTIAuthBackend
 from .saml import STANDARD_SAML_PROVIDER_KEY, get_saml_idp_choices, get_saml_idp_class
+from edx_toggles.toggles import SettingToggle
+
+# .. toggle_name: FEATURES['USE_LATEST_SAML_CONFIG']
+# .. toggle_implementation: SettingToggle
+# .. toggle_default: True
+# .. toggle_description: When enabled, the system uses the latest enabled SAML configuration with
+#    the same slug when authenticating users. When disabled, the system uses the directly
+#    referenced configuration.
+# .. toggle_warning: Disabling this may affect authentication for users if providers have
+#    been updated with newer configurations.
+# .. toggle_use_cases: opt_out
+# .. toggle_creation_date: 2025-06-10
+# .. toggle_target_removal_date: 2026-06-10
+# .. toggle_tickets: ARCHBOM- 334
```

Comment:
Let's not include private tickets in public code or commit messages. Having the link in the PR description is enough, and we can state that it is a private ticket.
```suggestion
```